tests/unit/modules/network/ios/test_ios_user.py

```diff
@@ -156,3 +156,31 @@ def test_ios_user_set_sshkey_multiple(self):
         ]
         result = self.execute_module(changed=True, commands=commands)
         self.assertEqual(result["commands"], commands)
+
+    def test_add_hashed_password(self):
+        hashed_password_val = "$9$.h8H2hF/PKwM2E$Sqx1m7o.sEEZbOPelcBFU2tCxc00j3Ukl3Yz9FMK7SE"
```

Comment:
security would complain about this, we should take it out or figure out a way to add something that is not really hashed!